src/components/AddTodoProject/AddTodoProject.js

```diff
@@ -0,0 +1,113 @@
+import StyledForm from '../styled-components/StyledForm';
+import StyledSection from '../styled-components/StyledSection';
+import StyledButton from '../styled-components/StyledButton';
+import StyledLabel from '../styled-components/StyledLabel';
+import StyledInput from '../styled-components/StyledInput';
+import StyledTextarea from '../styled-components/StyledTextarea';
+import StyledArticle from '../styled-components/StyledArticle';
+import StyledDiv from '../styled-components/StyledDiv';
+import MySVG from '../MySVG';
+import { useState } from 'react';
+import useStore from '../../hooks/useStore';
+
+export default function AddTodoProject() {
+	const [todonameValue, setTodonameValue] = useState('');
+	const [todonoteValue, setTodonoteValue] = useState('');
+	const [radio, setRadio] = useState('');
+	const addTodo = useStore(state => state.addTodo);
+
+	return (
+		<StyledForm
+			onSubmit={event => {
+				event.preventDefault();
+				addTodo(todonameValue, todonoteValue, radio);
+				setTodonameValue('');
+				setTodonoteValue('');
+				setRadio('');
+			}}
+		>
+			<StyledSection variant="big-grey">
+				<MySVG
+					variant="cancel"
+					position="absolute"
+					size="2.6rem"
+					top="-3.5rem"
+					right="0.5rem"
+				/>
+				<StyledLabel variant="default" htmlFor="todoname">
+					Todoname
+				</StyledLabel>
+				<StyledInput
+					required
+					variant="full"
+					id="todoname"
+					name="todoname"
+					type="text"
+					value={todonameValue}
+					onChange={event => {
+						setTodonameValue(event.target.value);
+					}}
+				/>
+				<StyledArticle variant="flexcenter">
+					<StyledTextarea
+						variant="addtodonote"
+						value={todonoteValue}
+						onChange={event => {
+							setTodonoteValue(event.target.value);
+						}}
+					/>
+				</StyledArticle>
+				<StyledLabel variant="toplabel" htmlFor="complexity">
+					Schwierigkeit
+				</StyledLabel>
+				<StyledArticle variant="flexcenter">
+					<StyledInput
+						variant="complexity"
+						type="radio"
+						name="complexity"
+						id="easy"
+						value="easy"
+						checked={radio === 'easy'}
+						onChange={event => {
+							setRadio(event.target.value);
+						}}
+					/>
+					<StyledLabel variant="complexity" htmlFor="easy">
+						<StyledDiv variant="green-complex" />
```

Comment:
Here you could use a better wording for the component, for example `ComplexityDisplay`